src/panels/config/tags/dialog-tag-detail.ts

```diff
@@ -172,7 +174,6 @@ class DialogTagDetail
 
   protected override firstUpdated(changedProps: PropertyValues): void {
     super.firstUpdated(changedProps);
-    this._generateQR();
   }
```

Comment:
Remove this

Reply:
I swear I did that locally 🤯 ... how did it come back?